### DISCUSSION TITLE
Implement `sqlite3_total_changes`

### DISCUSSION THREAD

**Discussion by gismofx** [rhashimoto/wa-sqlite#206] [Ideas]
I would like to use `sqlite3_total_changes` when executing multiple statements within a transaction. Can this be implemented?

As a work-around, I am cumulating `changes` function after executing each statement.

**Comment by schickling** [comment]
Should be easy enough to expose yourself in a fork and possibly @rhashimoto would be open to a PR for this.

**Comment by rhashimoto** [comment]
> As a work-around, I am cumulating changes function after executing each statement.

This missing function is similar to [`sqlite3_clear_bindings()`](https://github.com/rhashimoto/wa-sqlite/discussions/186): there is no wrapper function, the underlying C function is exported by the WebAssembly module (the argument to SQLite.Factory() creating the API wrappers), and the argument and return value are simple integers. So perhaps a more convenient workaround would be to call `module._sqlite3_total_changes(db)`.

I'll put adding a wrapper on my TODO list at low priority. Or as @schickling suggests, I'd be open to a PR (with a test).
